scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -233,7 +233,7 @@ object ScanCode : LocalScanner() {
         }
 
         errors.clear()
-        errors += mappedErrors.distinct()
+        errors += mappedErrors.distinctBy { it.message }
```

Comment:
You'd also need to do that for `mapTimeoutErrors()`.

Reply:
Done.